eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/equiz.highlightFilter.js

```diff
@@ -0,0 +1,15 @@
+﻿(function (angular) {

```

Comment:
Rename the file according to https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming